### DISCUSSION TITLE
Limit per checkout

### DISCUSSION THREAD

**Discussion by tanujrawat95** [saleor/saleor#12104] [General]
Hello everyone, I have made a product with variants and I can add them to cart. But I am facing an issue. I can only add 50 items per checkout and cannot go further than that. I dont want a limit per checkout as the product is digital product. 
Please help me resolve the issue.
I have also set the limit to none so that there wont be any limit.
![image](https://user-images.githubusercontent.com/66406059/220852694-5a63fbaf-79f5-412c-a922-685cf7917424.png)

**Comment by korycins** [comment]
If you do not specify a limit per checkout for a particular product, the general shop limit will be applied. 
If you do not want to impose a limit on a specific product, I suggest setting a large number that will accommodate your needs. 
If all of your products are digital and you want to eliminate restrictions altogether, you can set the global shop limit to a large number. To do this, go to the dashboard -> configuration -> site settings.

**Reply by tanujrawat95** [reply]
Hi @korycins , Thanks for the answer. I found one option in site setting with name "Checkout limits". I tried setting it up at higher number but its not working as well. 
I increased the limit on variants level and disable "track inventory" option, so its working for now.
